docs/csharp/programming-guide/concepts/async/task-asynchronous-programming-model.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "The Task Asynchronous Programming Model (TAP) with async and await (C#)"
+description: Learn about when and how to use async programming, a simplified approach to asynchronous programming in C#.
```

Comment:
add clarification:

```suggestion
description: Learn about when and how to use Task-bassed async programming, a simplified approach to asynchronous programming in C#.
```

Reply:
@BillWagner ? "task-based" - and should it be initial cap on "Task"?

Reply:
I think it should be "Task-based" to emphasize the name of the class. (Great catch).